### DISCUSSION TITLE
Simulation graph of camera position

### DISCUSSION THREAD

**Discussion by georgezh9617** [krikienoid/flagwaver#83] [General]
Wonderful update of camera control!
There's a thought:
It would be clearer for users if adding a graph in the wiki, and maybe add a simulation graph of camera position in the page too. (I'm a little unsure about what is r, θ, and φ, hence didn't put them into the image)
(This is the first discussion I posted. If the language I use is rude, sorry in advance)
![1](https://user-images.githubusercontent.com/128547250/226750864-6dc1719f-1a76-48ad-88ec-fbbaedc3ea2e.png)
![2](https://user-images.githubusercontent.com/128547250/226750871-615a27f9-accb-4834-968c-1ec3013bdb04.png)

**Comment by krikienoid** [comment]
Hi! Are you suggesting a change to the wiki or a change to the UI?

The r, θ, and φ inputs are just a way to manually set the same properties that you would normally control by using dragging and zooming on the screen. I don't know of an easy way to explain what these variables actually are without a lot of in-depth math talk but this Wikipedia page article is a good start: https://en.wikipedia.org/wiki/Spherical_coordinate_system

**Reply by georgezh9617** [reply]
Yes, a suggestion on wiki and UI, but still just a very brief and immature idea.
Thanks for providing the wikipedia page!
